x-pack/plugins/apm/public/components/shared/charts/helper/get_alert_annotations.test.tsx

```diff
@@ -23,28 +33,26 @@ const theme = ({
 } as unknown) as EuiTheme;
 const alert: Alert = {
   'rule.id': ['apm.transaction_duration'],
-  'kibana.rac.alert.evaluation.value': [2057657.39],
+  [ALERT_EVALUATION_VALUE]: [2057657.39],
```

Comment:
@ecezalp I added these just for the `kibana.alert` fields for now, since they are most likely to change. Will eventually add all of them, but maybe in the next PR?